src/org/usfirst/frc/team4915/steamworks/subsystems/Intake.java

```diff
@@ -42,10 +42,12 @@ public void setIntake(boolean onOff)
         {
             if (onOff)
             {
+                m_logger.info("Turn the motor on");
```

Comment:
yes - this could go either way depending on what we're after.  As long as it doesn't spew repeatedly I think info is good for now.